includes/class-wc-calypso-bridge-woocommerce-admin.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Contains customizations for WooCommerce Admin
+ *
+ * @package WC_Calypso_Bridge/Classes
+ * @since   1.0.17
+ * @version 1.0.0
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC Calypso Bridge WooCommerce Admin
+ */
+class WC_Calypso_Bridge_WooCommerce_Admin {
+	/**
+	 * Class instance.
+	 *
+	 * @var WC_Calypso_Bridge_WooCommerce_Admin instance
+	 */
+	protected static $instance = false;
+
+	/**
+	 * Get class instance
+	 */
+	public static function factory() {
+		if ( ! self::$instance ) {
+			self::$instance = new self();
+		}
+		return self::$instance;
+	}
+
+	/**
+	 * Initialize the class
+	 */
+	public function init() {
+		if ( ! apply_filters( 'wc_calypso_bridge_allow_wca_devdocs', false ) ) {
+			$this->remove_devdocs_menu_item();
+		}
+	}
+
+	/**
+	 * Remove the Dev Docs menu item unless allowed by the `wc_admin_get_feature_config` filter.
+	 */
+	public function remove_devdocs_menu_item() {
+		if ( ! function_exists( 'wc_admin_get_feature_config' ) ) {
+			/**
+			 * Override the default local configuration file generated by WC Admin.
+			 */
+			function wc_admin_get_feature_config() {
```

Comment:
It seems like this will override our feature config.  Might be dangerous since certain items are not yet ready for the wild (like onboarding).

Reply:
I think a better way to do this would be add a filter to WC Admin so that the filter can be used here to remove the feature.

Reply:
Added in https://github.com/woocommerce/woocommerce-admin/pull/2689